GoogleContainerTools/skaffold
# refactor: replace statik with embed

Fixes: #7235 

**Description**
 - remove statik mod and related code
 - rename statik-generate.sh to embedded-files-generate.sh and move to generated files to fs/*_generated folders so we can embed those files into output binary. 
 - update MakeFile accordingly. 
 - read files from the embedded fs in schema/print, credit/export, instrumentations/export method.
 - create fs diretory to hold embedded files(licenses, schemas) generated from script. 
 - refactor credits/export method to use io/fs.WalkDir to traversal credit director.
 - update tests. 
 
**Binary size result**
  - before (86,850,656 bytes (86.9 MB on disk))
  - after (91,637,424 bytes (91.6 MB on disk)) 
  Not sure why it's increased 

**Test Plan**
This should not break the exiting behavior.. related testing
  - run ``make`` should collect license and schemas by default
  - run ``out/skaffold credits`` should be export licenses successfully run ``out/skaffold schema get skaffold/v1`` should export skaffold/v1 schema
  - update a source file then re-run make should not trigger license and schemas collecting
  - run ``make clean`` then and ``make `` should trigger license and schemas collecting
  - run ``Local=true make`` should skip license and schemas collecting

**Follow up Issues**
 - #7343 investigate options for replacing golint
